src/lib/remark-plugins/rewrite-tutorial-links/__tests__/handle-tutorial-link.test.ts

```diff
@@ -0,0 +1,100 @@
+import { SectionOption } from 'lib/learn-client/types'
+import { handleTutorialLink } from '../utils'
+
+const TEST_TUTORIAL_SLUG = 'product/tutorial'
+const MOCK_TUTORIAL_MAP = {
+	'product/tutorial': '/product/tutorials/collection/tutorial',
+}
+
+const testEachCase = (cases: string[][]) => {
+	test.each(cases)(
+		'handleTutorialLink(%p) returns %p',
+		(input: string, expectedOutput: string) => {
+			expect(handleTutorialLink(input, MOCK_TUTORIAL_MAP)).toBe(expectedOutput)
+		}
+	)
+}
+
+describe('handleTutorialLink', () => {
```

Comment:
We're almost onboarded all products into beta but should we add some tests to make sure that only references to collections and tutorials with the product in beta will be rewritten as relative?